### PR TITLE
Bugfix: restore related document IDs in Bibliography resources

### LINKED ISSUE #908: related_document_ssim should show related druid in Bibliography resources
#848 (adding a link to `Page details` manuscript) affected how Bibliography resources display `related_document_ssim`. 

## Before
screen shot 2017-11-16 at 3 35 48 pm

## After (bug)
screen shot 2017-11-16 at 3 33 23 pm

### PR DESCRIPTION
**Pull request by camillevilla**
Fixes #908 

This PR restores expected behavior to the `related_document_ssim` / Manuscript field for Bibliography resources. 

## Before
screen shot 2017-11-16 at 3 33 23 pm

## After
linked_druid

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -51,13 +51,13 @@ def table_of_contents_separator(options = {})
...
 
   def manuscript_link(options = {})
-    druid = options[:value]
+    druid = options[:value][0]
     document = options[:document]
     ms_number = document['manuscript_number_tesim']
-    return if druid.blank? || ms_number.blank?
-    return druid if document['format_main_ssim'] != ['Page details']
+    return if druid.blank?
+    return druid if (document['format_main_ssim'] != ['Page details']) || ms_number.blank?
```
Comment (mejackreed) [inline app/helpers/application_helper.rb]: Should we try to return a link to the druid?

Comment (camillevilla) [reply app/helpers/application_helper.rb]: Sure, I can give that a swing.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -52,13 +52,15 @@ def table_of_contents_separator(options = {})
...
+    return if options[:value].blank?
+    druid = options[:value][0]
     document = options[:document]
-    ms_number = document['manuscript_number_tesim']
-    return if druid.blank? || ms_number.blank?
-    return druid if document['format_main_ssim'] != ['Page details']
     title = document['title_full_display']
-    title = title.include?(':') ? title.partition(':')[2] : ms_number[0]
-    link_to title, spotlight.exhibit_solr_document_path(current_exhibit, druid[0])
+    link_title = if (document['format_main_ssim'] == ['Page details']) && title.include?(':')
```
Comment (jkeck) [inline app/helpers/application_helper.rb]: We have an accessor on `SolrDocument` that can tell us if the item is a canvas/page details object which might be useful here. https://github.com/sul-dlss/exhibits/blob/0779d173ea2d0861827e252aed21d76fc4e62505/app/models/concerns/canvas_concern.rb#L6-L8

Comment (camillevilla) [reply app/helpers/application_helper.rb]: 👍 sweet

**Comment by camillevilla** [comment]
alrighty @mejackreed @jkeck! This one's ready for eyeballs 👀

**Pull request merged**